### ISSUE TITLE
using EXT:frontend_editing with EXT:vhs and clicking on "show hidden content" an exception appears

### ISSUE THREAD

**Issue by erredeco** [FriendsOfTYPO3/frontend_editing#401]
[Please look at https://github.com/FluidTYPO3/vhs/issues/1617 for reference]

I found this error when using EXT:frontend_editing and clicking on the "show hidden content element" icon:

<img width="362" alt="Schermata 2020-01-23 alle 23 25 21" src="https://user-images.githubusercontent.com/3647646/73065851-d6f28c80-3ea4-11ea-9a53-b059d4af5321.png">

```
(1/1) #1384611413 TYPO3Fluid\Fluid\Core\ViewHelper\Exception
No record was found. The "record" or "uid" argument must be specified.

in /Siti/typo3.9.demo.it/public/typo3conf/ext/vhs/Classes/Utility/ErrorUtility.php line 25

     */
    public static function throwViewHelperException($message = null, $code = null)
    {
        if (version_compare(TYPO3_version, '8.0', '>=')) {
            throw new \TYPO3Fluid\Fluid\Core\ViewHelper\Exception($message, $code);
        }
        throw new \TYPO3\CMS\Fluid\Core\ViewHelper\Exception($message, $code);
    }
}

at FluidTYPO3\Vhs\Utility\ErrorUtility::throwViewHelperException('No record was found. The "record" or "uid" argument must be specified.', 1384611413)

...
```

Here is the full stack:

[vhs-issue.pdf](https://github.com/FriendsOfTYPO3/frontend_editing/files/4108167/vhs-issue.pdf)

The problem is actually caused by the usage of `<v:page.resources.fal>` in this way:

```
<v:page.resources.fal table="pages" field="nav_thumbnail" uid="{item.uid}" as="images">{images.0.alternative}</v:page.resources.fal>
```
inside one of the page partials: EXT:bhsiteconf/Resources/Private/Partials/Page/Megamenu.html:

```
<html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers"
      xmlns:v="http://typo3.org/ns/FluidTYPO3/Vhs/ViewHelpers"
      data-namespace-typo3-fluid="true">

<div class="accordion-group">
    <v:menu entryLevel="0" levels="1" expandAll="1" >
      
        <f:for each="{menu}" as="item">

        <f:variable name="alt_thumbnail">
            <v:page.resources.fal table="pages" field="nav_thumbnail" uid="{item.uid}" as="images">{images.0.alternative}</v:page.resources.fal>
        </f:variable>


        <f:variable name="description_thumbnail">
            <v:page.resources.fal table="pages" field="nav_thumbnail" uid="{item.uid}" as="images">{images.0.description}</v:page.resources.fal>
        </f:variable>



            <f:if condition="{item.hasSubPages}">
                <div class="collapse dropdown-mega collapsetoggle" id="a{item.uid}" aria-labelledby="al-{item.uid}">
                    <div class="container--extrawide">
                        <f:if condition="{item.nav_thumbnail}">
                            <div class="dropdown-mega__image-text-container">
                                <img class="dropdown-mega__image img-fluid" src="{f:uri.image(src:item.nav_thumbnail,treatIdAsReference:1,cropVariant:'default',width:'335')} " alt="{alt_thumbnail}" />

                                <f:if condition="{description_thumbnail}">
                                    <div class="dropdown-mega__text">
                                        <p>{description_thumbnail}</p>
                                    </div>                                    
                                </f:if>
                            </div>
                        </f:if>
   
                        <div class="dropdown-mega__menu-container">
                            <v:menu pageUid="{item.uid}" includeSpacers="1" classActive="menu__item--active" classCurrent="menu__item--active" classHasSubpages="">    
                                <ul class="menu menu--mega">  
                                    <f:for each="{menu}" as="subItem">
                                        <f:if condition="{subItem.doktype} == 199">
                                            <f:then>
                                                </ul><!--separatore--><ul class="menu menu--mega">  
                                            </f:then>
                                            <f:else>
                                                <li class="menu__item {subItem.class}">
                                                    <f:link.page pageUid="{subItem.uid}" title="{subItem.linktext}">{subItem.linktext}</f:link.page>
                                                </li>
                                            </f:else>
                                        </f:if>
                                    </f:for>           
                                </ul>
                            </v:menu>  
                        </div>
                    </div>
                </div>
            </f:if>
        </f:for>
    </v:menu>
</div>

</html>
```

Changing it to 
```
            <v:page.resources.fal table="pages" field="nav_thumbnail" record="{item}" as="images">{images.0.alternative}</v:page.resources.fal>

```

Actually solves the problem;  I open this issue because this might be just a workaround; please look at https://github.com/FluidTYPO3/vhs/issues/1617 for reference and to the useful comments of Claus Due, who has my eternal gratitude for digging into this problem, like: https://github.com/FluidTYPO3/vhs/issues/1617#issuecomment-578092094 .

**Comment by mabolek** [comment]
Accidentally closed this issue instead of #400.

**Comment by MattiasNilsson** [comment]
The feature "The hidden content" will be removed in v2 of frontend_editing. This is due to the fact that we render stuff in the backend now, where this #toggling" are much easier.
